src/libs/ReportUtils.js

```diff
@@ -765,62 +798,49 @@ function getIcons(report, personalDetails, defaultIcon = null, isPayer = false)
             type: CONST.ICON_TYPE_AVATAR,
         };
 
+        if (isWorkspaceThread(report)) {
+            const workspaceIcon = getWorkspaceIcon(report);
+            return [actorIcon, workspaceIcon];
+        }
         return [actorIcon];
     }
     if (isDomainRoom(report)) {
-        result.source = Expensicons.DomainRoomAvatar;
-        return [result];
-    }
-    if (isAdminRoom(report)) {
-        result.source = Expensicons.AdminRoomAvatar;
-        return [result];
-    }
-    if (isAnnounceRoom(report)) {
-        result.source = Expensicons.AnnounceRoomAvatar;
-        return [result];
+        const domainName = Str.extractEmailDomain(report.ownerEmail);
+        const policyExpenseChatAvatarSource = lodashGet(allPolicies, [`${ONYXKEYS.COLLECTION.POLICY}${report.policyID}`, 'avatar']) || getDefaultWorkspaceAvatar(domainName);
+        const domainIcon = {
+            source: policyExpenseChatAvatarSource,
+            type: CONST.ICON_TYPE_WORKSPACE,
+            name: domainName,
+        };
+        return [domainIcon];
```

Comment:
this change seems to be the cause of the failing test, but I am not yet sure why

Reply:
seems like doing anything to report.ownerEmail (i.e, substring, etc, even if you don't reference it) caused the test to fail